### PR TITLE
use goflags

### PR DESCRIPTION
**Pull request by parrasajad**
## Description
This PR adds support for `goflags`

### CODE REVIEW

File: internal/runner/options.go [hunk 1]
```diff
@@ -11,42 +10,51 @@ import (
...
+	Config         string                        // Config is the location of the config file.
+	Output         string                        // Output is the file to write found results too.
+	ExcludePrivate bool                          // ExcludePrivate excludes private IPs from results
+	Provider       goflags.NormalizedStringSlice // Provider specifies what providers to fetch assets for.
+	ProviderConfig string                        // ProviderConfig is the location of the provider config file.
 }
 
-var defaultConfigLocation = path.Join(userHomeDir(), "/.config/cloudlist/config.yaml")
+var (
+	defaultConfigLocation         = path.Join(userHomeDir(), "/.config/cloudlist/config.yaml")
```
Comment (Mzack9999) [inline internal/runner/options.go:35]: please use `filepath`

File: internal/runner/options.go [hunk 2]
```diff
@@ -11,42 +10,51 @@ import (
...
+	defaultProviderConfigLocation = path.Join(userHomeDir(), "/.config/cloudlist/provider-config.yaml")
```
Comment (Mzack9999) [inline internal/runner/options.go:36]: please use `filepath`

File: internal/runner/runner.go [hunk 3]
```diff
@@ -183,3 +188,12 @@ func (r *Runner) Enumerate() {
 		}
 	}
 }
+
+func Contains(s []string, e string) bool {
+	for _, a := range s {
+		if reflect.DeepEqual(a, e) {
```
Comment (Mzack9999) [inline internal/runner/runner.go:194]: Shouldn't `if strings.EqualFold(a, e) {` have the same result?

**Review by Mzack9999 — CHANGES_REQUESTED** [review]
Left some comments about `filepath` and comparison

**Review by ehsandeep — CHANGES_REQUESTED** [review]
- [ ] adding mssing cli flag for `id` filter

```
   -id string[]            ids to fetch assets from
```
- [ ] automatic migration of current config to provider config https://github.com/projectdiscovery/cloudlist/issues/39

**Review by ehsandeep — APPROVED** [review]

**Pull request merged**